readme.md

```diff
@@ -73,3 +73,36 @@ bot.once('spawn', () => {
 * Dynamic harvest/mining paths
 * Sprint
 * Actual move speed per block
+
+## Wiki(WIP)
+Considering there are a lot of deep changes that are being worked on, it could take some time before it's done
+
+Also, **for now**, there is only the `pathfinder` module, `movements` and `goals` still need to be done
+
```

Comment:
Might be better to add atleast basic text for those before merging. Even if it's just listing what the goals/movement functions are.

Reply:
while looking at movements.js i was wondering, is it necessary to include it in the docs?
also, for goto.js, should i document that too?